lib/adapter/freshheads/MinimizationAdapter.ts

```diff
@@ -22,12 +22,7 @@ export default class MinimizationAdapter implements Adapter {
         // For webpack@5 you can use the `...` syntax to extend existing minimizer (i.e. `terser-webpack-plugin`)
         // So Javascript will be minimized by default
         new OptimizationAdapter({
-            minimizer: [
-                `...`,
-                new CssMinimizerPlugin({
-                    sourceMap: true,
```

Comment:
Default tegenwoordig? Of gewoon niet meer nodig om hier aan te geven?

Reply:
Is komen te vervallen, ze gaan nu uit van de centrale webpack sourcemap setting.

Reply:
ah, beter!